packages/ui/__stories__/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useRef, useState } from 'react'
+import { v4 as uuid } from 'uuid'
+import { Radio } from '../src/Radio'
+import { RadioGroup } from '../src/RadioGroup'
+import { Form, Formik } from 'formik'
+import { RadioGroupFieldFormik } from '../src/RadioGroupFormik'
+import { RadioFieldFormik } from '../src/RadioFormik'
+import { logger } from '@hazelcast/services'
+
+export default {
+  title: 'Components/RadioGroup',
+  component: RadioGroup,
+}
+
+export const RadioGroupDefault = () => {
+  const idRef = useRef(uuid())
+  const [value, setValue] = useState<string>('test')
+  return (
+    <RadioGroup>
+      <Radio
+        name={idRef.current}
+        value="test2"
+        label="Test 2"
+        disabled
+        checked={value === 'test2'}
+        onChange={({ target: { value } }) => setValue(value)}
+      />
+      <Radio
+        name={idRef.current}
+        value="test"
+        label="Test"
+        checked={value === 'test'}
+        onChange={({ target: { value } }) => setValue(value)}
+      />
+    </RadioGroup>
+  )
+}
+
+export const RadioGroupWithDescriptionAndError = () => {
+  const idRef = useRef(uuid())
```

Comment:
nit: Couldn't we use a simple string top-level constant in the beginning of the file instead?

Reply:
@aigoncharov since this ID is used as a name to a native HTML radio input, we don't want different stories to re-use it because it'd link different stories together.